Added Reset Cookie functionality

Rutuj-Runwal opened the pull request:
Reset cookie functionality is added.
Now the user can click on the "Reset Cookies" button to remove the cookies and make the website reset to its original state.

In reference to issue no #29 

Demo link: https://drive.google.com/file/d/1y0Oy9Nx0OidE6gOFbcYzqdEJcc_0QRfZ/view?usp=sharing

akhil-aggarwal commented:
Please correct the alignment of the button. Also, see if we can place the button somewhere else although its optional.

Rutuj-Runwal commented:
@akhil-aggarwal I have corrected the alignment of Reset Cookies button and added a confirmation dialog before resetting the cookies.
![image](https://user-images.githubusercontent.com/59436520/193447353-0789d915-e1c2-4681-bdf3-2bc06d44f4c1.png)
